extensions/dicom-seg/src/commandsModule.js

```diff
@@ -0,0 +1,21 @@
+// "actions" doesn't really mean anything
+// these are basically ambigous sets of implementation(s)
+const actions = {
+  doGoofyStuff: () => {
```

Comment:
I'm 110% sure this is placeholder that can be removed.

Reply:
Yupp. Looks like we can remove the commandsModule entirely 👍

Reply:
Removed. 👍